### DISCUSSION TITLE
Anyone with a SD 855, or lower?

### DISCUSSION THREAD

**Discussion by xXxNobuoxXx** [skyline-emu/skyline#915] [General]
Do you think one day Skyline (I know it's still early) will be able to play 3D games on a Snapdragon 855 or lower? Or would I need a more powerful phone? I can run some games at 40 fps or 60 fps but others not so much.

**Comment by ErickSRWfan** [comment]
I have a snapdragon 845

**Comment by Alex1903M** [comment]
I have a SD 835

**Comment by finr0y** [comment]
The devs have around 845/855/865 so I think they realize these constraints, just give them time to fully implement the things that they need to and optimize the emulator.

**Reply by xXxNobuoxXx** [reply]
Yeah I figured that.. just hoping one day to play Skyrim or BoTW and some others on my android at work instead on carrying a big bulky Nintendo Switch in my pocket 😜

**Reply by finr0y** [reply]
Maybe one day in the future 😉

**Comment by frans4080** [comment]
> Do you think one day Skyline (I know it's still early) will be able to play 3D games on a Snapdragon 855 or lower? Or would I need a more powerful phone? I can run some games at 40 fps or 60 fps but others not so much.

most likely you will be able to play some 3d games that aren't that aren't very power hungry.
